src/dropWhile.js

```diff
@@ -1,6 +1,7 @@
 var _curry2 = require('./internal/_curry2');
 var _dispatchable = require('./internal/_dispatchable');
 var _xdropWhile = require('./internal/_xdropWhile');
+var slice = require('./slice');
```

Comment:
In fact, it's the year of the eclipse.  We can use that excuse until January, I believe. :smile: